Formula/percona-xtrabackup.rb

```diff
@@ -0,0 +1,77 @@
+class PerconaXtrabackup < Formula
+  desc "Open source hot backup tool for InnoDB and XtraDB databases"
+  homepage "https://www.percona.com/software/mysql-database/percona-xtrabackup"
+  url "https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.2/source/tarball/percona-xtrabackup-2.4.2.tar.gz"
+  sha256 "faeac6f1db4a1270e5263e48c8a94cc5c81c772fdea36879d1be18dcbcd1926e"
+
+  option "without-docs", "Build without man pages (which requires python-sphinx)"
+  option "without-dbd-mysql", "Build without bundled Perl DBD::mysql module, to use the database of your choice."
+
+  depends_on "cmake" => :build
+  depends_on "libev" => :build
+  depends_on "sphinx-doc" => :build if build.with? "docs"
+  depends_on :mysql => :build if build.with? "dbd-mysql"
```

Comment:
I'm not really sure we need the option at all, to be honest. It's pretty much already saying what our advice is in these situations:

* If you want to use the bottle, which will be built against `mysql`, you're good.
* If you want to use another formula that satisfies the `mysql` requirement you need to build from source.

I think I still prefer just disabling the bottle here, and making `docs` optional rather than recommended consequently. Open to thoughts though.